staging/src/k8s.io/client-go/discovery/discovery_client.go

```diff
@@ -55,6 +56,12 @@ const (
 
 	// defaultBurst is the default burst to be used with the discovery client's token bucket rate limiter
 	defaultBurst = 300
+
+	acceptV1 = runtime.ContentTypeJSON
+	// Aggregated discovery content-type (currently v2beta1)
+	acceptV2Beta1 = runtime.ContentTypeJSON + ";" + "g=discovery.k8s.io,v=v2beta1,as=APIGroupDiscoveryList"
```

Comment:
Accept entries are separated by commas, components of an entry are separated with semi-colon. Here you have a little bit of a mix of both:
```suggestion
	acceptV2Beta1 = runtime.ContentTypeJSON + ";g=discovery.k8s.io;v=v2beta1;as=APIGroupDiscoveryList"
```

Reply:
Fixed